job-service/src/main/java/com/hpe/caf/services/job/api/JobsGet.java

```diff
@@ -71,6 +93,11 @@
                 }
             }
 
+            List<String> labelValues = null;
+            if(!StringUtils.isEmpty(labelExists)) {
```

Comment:
seems a little heavyhanded for us to add the spring-core dependency just to check if a String is null or empty. Can we not just check this ourselves? It seems like a waste for such a tiny utility function.